opennms-base-assembly/src/main/filtered/etc/org.opennms.plugin.elasticsearch.rest.forwarder.cfg

```diff
@@ -0,0 +1,54 @@
+# org.opennms.plugin.elasticsearch.rest.forwarder.cfg
```

Comment:
I'll trim this file out, it is redundant with the admin doc descriptions of the fields and I think we've agreed before in discussions with @indigo423 that moving documentation out of the bundled config files and into the proper documentation is the best way to handle this.